snekde/snekde.py

```diff
@@ -1041,7 +1040,7 @@ def screen_paint():
     stdscr.addstr(mid_y, device_col, device_name, curses.A_REVERSE)
     if device_col >= 6:
         stdscr.addstr(mid_y, device_col - 6, "      ", curses.A_REVERSE)
-    for col in range(0,device_col - 6,5):
+    for col in range(0, device_col - 6, 5):
```

Comment:
Maybe we should run 'black' over the code? Looks like that finds a *lot* of places this code isn't following style conventions :-)

Reply:
Yes, I would be in favor of running Black over the code. [Black is available in the browser](https://black.now.sh/). But I guess it would also possible be possible to use configure GitHub to run some action to integrate Black on commits or pull requests.